### ISSUE TITLE
Screen is flickering

### ISSUE THREAD

**Issue by mbpictures** [NicosNicolaou16/Ink_Api_Compose#2]
Hi!

First of all: thank you very much for sharing your example! While it works perfectly when using in the emulator, the screen flickers when using on a real android device (Samsung Galaxy S20+). I get a lot of these errors:

```
2024-12-31 13:28:08.105 27825-27881 FrameEvents             com.nicos.ink_api_compose            E  updateAcquireFence: Did not find frame.
```

And sometimes these logs:
```
2024-12-31 13:28:08.092 27825-27825 Gralloc2                com.nicos.ink_api_compose            E  buffer descriptor contains invalid usage bits 0x100000000
2024-12-31 13:28:08.092 27825-27825 GraphicBufferAllocator  com.nicos.ink_api_compose            E  Failed to allocate (4 x 4) layerCount 1 format 1 usage 100000b00: -22
2024-12-31 13:28:08.092 27825-27825 AHardwareBuffer         com.nicos.ink_api_compose            E  GraphicBuffer(w=4, h=4, lc=1) failed (Invalid argument), handle=0x0
```

Can you experience this by yourself? Do you think it's related to the android ink API or to your implementation?

Thanks already!

**Comment by NicosNicolaou16** [comment]
Hello @mbpictures , I tried this example on two Samsung tablets and didn't encounter these errors on my side. These issues are most likely related to the Ink API, as it is still in the 1.0.0-alpha02 version, and Google is still working on it.

Thank you!

**Comment by mbpictures** [comment]
Thanks for your fast Response! I will crosspost it to the related issuetracker.

**Comment by NicosNicolaou16** [comment]
Hello @mbpictures, is it fixed?

**Comment by mbpictures** [comment]
Yes! It got fixed in the Version of the graphics library (has to be specified in gradle) and with alpha04 of the ink lib

**Comment by NicosNicolaou16** [comment]
Hello @mbpictures, I released a new version with this fix!

[v1.0.7](https://github.com/NicosNicolaou16/Ink_Api_Compose/releases/tag/1.0.7)
